### PR TITLE
feat: Added KYC expiration support

### PR DESCRIPTION
**Pull request by frol**
The [backend has been migrated](https://github.com/NEAR-DevHub/neardevhub-kyc-proxy/commit/3758c891a36261360a3e0d3de47e30d7064c9b66) to the new Shuttle infrastructure, so the URL has changed (the old URL is still active and runs the old version of the code).

The reported KYC status is now in SCREAMING_SNAKE_CASE and a new `EXPIRED` status is added.

I have also upgraded events-committee code (copied the whole component)

@race-of-sloths include

### CODE REVIEW

**Comment by race-of-sloths** [comment]
@frol Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!


[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://race-of-sloths.com/images/bot-banner-dark.svg">
    <source media="(prefers-color-scheme: light)" srcset="https://race-of-sloths.com/images/bot-banner-light.svg">
    Shows inviting banner with latest news.
</picture>](https://race-of-sloths.com/latest-news)

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/frol?type=bot&pr=NEAR-DevHub/neardevhub-bos/980&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/frol?type=bot&pr=NEAR-DevHub/neardevhub-bos/980&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/frol)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command. Alternatively, autoscoring [1,2] will be applied for this pull request

</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/frol)

For maintainers:
- Score pull requests that participate in the Race of Sloths and receive a reward
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

File: instances/devhub.near/widget/devhub/entity/proposal/VerificationStatus.jsx
Comment (Tguntenaar) [inline instances/devhub.near/widget/devhub/entity/proposal/VerificationStatus.jsx:34]: Verified is misspelled here

File: instances/events-committee.near/widget/devhub/entity/proposal/VerificationStatus.jsx
Comment (Tguntenaar) [inline instances/events-committee.near/widget/devhub/entity/proposal/VerificationStatus.jsx:34]: Once more here

File: instances/devhub.near/widget/devhub/entity/proposal/VerificationStatus.jsx
Comment (frol) [reply instances/devhub.near/widget/devhub/entity/proposal/VerificationStatus.jsx]: @Tguntenaar Good catch! I fixed it

**Review by Megha-Dev-19 — APPROVED** [review]
LGTM

**Comment by Megha-Dev-19** [comment]
@race-of-sloths score 3

**Comment by petersalomonsen** [comment]
@race-of-sloths score 3

**Comment by petersalomonsen** [comment]
should also add tests for this feature.

**Pull request merged**
